<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1581</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 13pt &quot;黑体&quot;;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>40</y>
      <width>241</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>13</pointsize>
      <weight>11</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#bcd1ca;&quot;&gt;God's perspective&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 88 13pt &quot;黑体&quot;;
</string>
    </property>
    <property name="title">
     <string>上帝的视角</string>
    </property>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;
border-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pause&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>暂停</string>
     </property>
    </widget>
    <widget class="QPushButton" name="restartButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>150</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Restart&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>重新开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="endButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;End&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>结束</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resumeButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Resume&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>继续</string>
     </property>
    </widget>
    <widget class="QPushButton" name="setButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Setting&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>人为干预</string>
     </property>
    </widget>
    <widget class="QPushButton" name="damageButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Natural Disaster&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>自然灾害</string>
     </property>
    </widget>
    <widget class="QPushButton" name="fineButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;favourable weather&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;黑体&quot;;</string>
     </property>
     <property name="text">
      <string>风调雨顺</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox2">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>530</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#bcd1ca;&quot;&gt;Others&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="title">
     <string>其他</string>
    </property>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>11</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 100 11pt &quot;微软雅黑&quot;;
</string>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
    </widget>
    <widget class="QPushButton" name="aboutButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;About us&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>关于我们</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exit&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 11pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1581</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="text">
    <string>Resume</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionEnd">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>End</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
